Source/Core/DolphinQt2/MenuBar.cpp

```diff
@@ -271,3 +309,58 @@ void MenuBar::InstallWAD()
 
   result_dialog.exec();
 }
+
+void MenuBar::PerformOnlineUpdate(const std::string& region)
+{
+  const int confirm = QMessageBox::question(
+      nullptr, tr("Confirm"), tr("Connect to the Internet and perform an online system update?"));
```

Comment:
Ah, I wasn't sure whether I should use `this` or not, as I thought it would cause it to be centered over the menu bar (since the this refers to the MenuBar, right?). I'll fix this.